### PR TITLE
binaryclk: added artwork to calendar and setting to show battery

### PR DESCRIPTION
**Pull request by shansou504**

### CODE REVIEW

File: apps/binaryclk/app.js [hunk 1]
```diff
@@ -53,11 +59,47 @@ function draw() {
...
+			}
+			g.fillRect(Math.floor(mgn/2) + gap + 2 * pos, mgn + gap, Math.floor(mgn/2) + gap + 2 * pos + Math.floor(bat * sq / 100), mgn + gap + sq);
+			if (g.theme.dark) {
+				g.setColor(-1);
+			} else {
+				g.setColor(1);
+			}
+			g.drawRect(Math.floor(mgn/2) + gap + 2 * pos, mgn + gap, Math.floor(mgn/2) + gap + 2 * pos + sq, mgn + gap + sq);
+		}
+		cnt =+ cnt;
```
Comment (bobrippling) [inline apps/binaryclk/app.js:99]: Did you mean something like this here?
```suggestion
		cnt += cnt;
```

or:
```suggestion
		cnt++;
```

**Review by bobrippling — COMMENTED** [review]
Looks good - one little spot

File: apps/binaryclk/app.js (hunk 1 above)
Comment (shansou504) [reply apps/binaryclk/app.js]: Should be updated now, thank you!

**Comment by bobrippling** [comment]
Nice, thanks!

**Pull request merged**